### PR TITLE
feat(cdsctl): allow many users on same url

### LINKED ISSUE #4379: cdsctl user login remains always on the first login
I've created a user "admin" and then I log to CDS via:
./cdsctl login --api-url http://localhost:8081 -u admin

Then I've created a new user "admin2" and I logged to CDS via:
./cdsctl login --api-url http://localhost:8081 -u admin2

But "./cdsctl user me" remains to user "admin".
I expected to move to user "admin2". 
Maybe I miss a deconexion or logout.

In fact, I don't know how to change user.

### PR DESCRIPTION
**Pull request by yesnault**
close #4379

Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

File: docs/content/hosting/ready-to-run/docker-compose.md [hunk 1]
```diff
@@ -59,8 +59,8 @@ After registration on UI, keep the password displayed, we will use it in next st
 
 - Login with cdsctl
 
-Please note that the version linux/amd64, darwin/amd64 and windows/amd64 use libsecret / keychain to store the CDS Password.
-If you don't want to use the keychain, you can select the version i386.
+*please note that the version linux/amd64, darwin/amd64 and windows/amd64 use libsecret / keychain to store the CDS Password.
+If you don't want to use the keychain, you can select the version `-nokeychain`
```
Comment (bnjjj) [inline docs/content/hosting/ready-to-run/docker-compose.md]:
```suggestion
If you don't want to use the keychain, you can select the version suffixed by `-nokeychain`
```

File: contrib/helm/cds/README.md [hunk 2]
```diff
@@ -181,7 +181,7 @@ $ chmod +x cdsctl
 ```
 
 *please note that the version linux/amd64, darwin/amd64 and windows/amd64 use libsecret / keychain to store the CDS Password.
-If you don't want to use the keychain, you can select the version i386*
+If you don't want to use the keychain, you can select the version `-nokeychain`
```
Comment (bnjjj) [inline contrib/helm/cds/README.md]:
```suggestion
If you don't want to use the keychain, you can select the version suffixed by `-nokeychain`
```

**Comment by ovh-cds** [comment]
CDS Report build-engine-cli-tools#10643.0 ✘  
* Build minimal
  * cdsctl linux ✘  
  * check SQL Files number ✔ 
  * Engine Build linux amd64 ✔ 
  * tools/debpacker ✔ 
  * tools/os-ansible-inventory ✔ 
  * Worker linux amd64 / 386 ✔

**Comment by yesnault** [comment]
will be done on https://github.com/ovh/cds/tree/feat/auth with context switch

**Pull request closed without merging**